user/enterprise/setting-up-travis-ci-enterprise.md

```diff
@@ -0,0 +1,162 @@
+and Docker.
+
+Each dedicated host or hypervisor (VMWare, OpenStack using KVM, or EC2) should
+run **Ubuntu 14.04** or **Ubuntu 16.04**, ideally using Linux 3.16 and have at least **16 gigs of
+RAM and 8 CPUs**.
+
+If you're running on EC2, we recommend the **c4.2xlarge** instance type for both **Platform** and **Worker**. We also recommend using an image that uses EBS for the root volume, as well as allocating 40 gigs of space to it. It is also recommended _not_ to destroy the volume on instance termination.
+
+For [high availability (HA)](/user/enterprise/high-availability/) configurations, you will also need:
+
+* [Redis](https://redis.io/),
+* [RabbitMQ](https://www.rabbitmq.com/)
+* [Postgres](https://www.postgresql.org/)
+
+You can also try services like [compose.com](https://compose.com/) if you would like these services hosted outside your organization.
+
+## 1. Setting up  Enterprise Platform virtual machine
+
+The Travis CI Enterprise Platform handles licensing, coordinates worker
+processes, and maintains the Enterprise user and admin dashboard. It must be
+installed on it's own machine instance, separate from that of the Travis CI
+Enterprise worker. We recommend using AWS' `c4.2xlarge` instance running
+Ubuntu 16.04 LTS as the underlying operating system.
+
+1. *On your virtual machine management platform*, create a Travis CI Platform Security Group.
+
+    If you're setting up your AMI for the first time you need to create
+    a Security Group. From the EC2 management console, create an entry for
+    each port in the table below:
+
+    | Port | Service         | Description                                                                  |
+    |:-----|:----------------|:-----------------------------------------------------------------------------|
+    | 8800 | Custom TCP Rule | This port is to access the admin dashboard for your Enterprise installation. |
+    | 5672 | Custom TCP Rule | For RabbitMQ Non-SSL.                                                        |
+    | 4567 | Custom TCP Rule | For RabbitMQ SSL.                                                            |
+    | 443  | HTTPS           | Web application over HTTPS access.                                           |
+    | 80   | HTTP            | Web application access.                                                      |
+    | 22   | SSH             | SSH access.                                                                  |
+
+1. If you're using a hostname and not just an IP address, this would be a time to configure it.
+
+1. *On your new virtual machine*, download and run the installation script:
+
+
+    ```
+    curl -sSL -o /tmp/installer.sh https://enterprise.travis-ci.com/install
+    sudo bash /tmp/installer.sh
+    ```
+
+3. *In your browser*, navigate to `https://<hostname>:8800` (your Enterprise
+installation's hostname, port 8800) to complete the setup:
+
+    a. Add a secure certificate / configure a trusted one [HTTPS Replicated page]  
```

Comment:
Do we need to indicate that this is a page managed by Replicated? Aren't all of these pages managed by Replicated?

Reply:
Everything that's living behind port 8800 is managed by Replicated. We configure it but Replicated does everything else. 

Hmm, stating that this is managed by Replicated, would that provide a benefit for users?

Reply:
Personally? I don't think so. As a customer I don't really care that Travis CI is using Replicated to manage the install. It's just extra noise, mentioning a thing that is not referenced anywhere else. 

The only way I could see it adding value is if we add some doc explaining how we use Replicated and link to the document, offering the user a clearer idea of exactly what they are installing so they can communicate more effectively when they need support. But I don't know if any customer would find that useful. 

If it were me as a customer? I wouldn't want to have to know about that. 🤷‍♂️

Reply:
Nothing to worry about, that is just an artifact from the discussion yesterday, I'll remove it tomorrow when I go over all of the feedback